std/bitmanip.d

```diff
@@ -3536,6 +3537,7 @@ unittest
         T     = The integral type to convert the first $(D T.sizeof) bytes to.
         endianness = The endianness to write the bytes in.
         range = The range to append to.
+        value = The value to append.
```

Comment:
s/append/_append/

The above existing line also makes the same mistake (this is such a misfeature of DDoc...)